src/Components/CaseSummaryCard/styles.ts

```diff
@@ -0,0 +1,49 @@
+import { StyleSheet } from 'react-native';
+
+export default StyleSheet.create({
```

Comment:
Let's add a shadow to the card/button as seen on the Figma!

Reply:
Do we want to move forward with this?

Reply:
Yes! Let's try tackling this with `shadow` attributes for the `TouchableOpacity`. Looking at [this](https://stackoverflow.com/questions/50162879/create-raised-or-shadow-effect-on-touchableopacity-react-native), it seems like we just need to use a combination of IOS and Android props but I will do more exploring